taichi/codegen/wasm/codegen_wasm.h

```diff
@@ -23,6 +23,8 @@ class KernelCodeGenWASM : public KernelCodeGen {
   LLVMCompiledData compile_task(
       std::unique_ptr<llvm::Module> &&module = nullptr,
       OffloadedStmt *stmt = nullptr) override;  // AOT Module Gen
+
+  std::vector<LLVMCompiledData> compile_kernel_to_module() override;
```

Comment:
returned value isn't really `module` ;) considering changing to a better name?

Reply:
After refactoring in the future PRs, this function will return a `LLVMCompiledKernel` which contains only a module and the names of the offloaded task functions.